apps/ops/models/job.py

```diff
@@ -349,7 +354,8 @@ def get_runner(self):
             runner = PlaybookRunner(
                 self.inventory_path,
                 self.current_job.playbook.entry,
-                self.private_dir
+                self.private_dir,
+                extra_vars=extra_vars,
             )
         elif self.current_job.type == Types.upload_file:
             job_id = self.current_job.id
```

Comment:
There are no reported issues with the provided code. It appears to be functioning correctly based on the knowledge cutoff and the current date. However, if there's anything specific you'd like reviewed that wasn't covered by these checks, feel free to ask!